src/pages/workspace/WorkspaceSettingsCurrencyPage.js

```diff
@@ -0,0 +1,106 @@
+import React, {useState, useMemo, useCallback} from 'react';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import useLocalize from '../../hooks/useLocalize';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithBackButton from '../../components/HeaderWithBackButton';
+import SelectionList from '../../components/SelectionList';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import withPolicy, {policyDefaultProps, policyPropTypes} from './withPolicy';
+import * as Policy from '../../libs/actions/Policy';
+
+const propTypes = {
+    /** Constant, list of available currencies */
+    currencyList: PropTypes.objectOf(
+        PropTypes.shape({
+            /** Symbol of the currency */
+            symbol: PropTypes.string,
+        }),
+    ),
+    ...policyPropTypes,
+};
+
+const defaultProps = {
+    currencyList: {},
+    ...policyDefaultProps,
+};
+
+function WorkspaceSettingsCurrencyPage({currencyList, policy}) {
+    const {translate} = useLocalize();
+    const [searchText, setSearchText] = useState('');
+
+    const getDisplayText = useCallback((currencyCode, currencySymbol) => `${currencyCode} - ${currencySymbol}`, []);
+
+    const {sections, initiallyFocusedOptionKey} = useMemo(() => {
+        const trimmedText = searchText.trim().toLowerCase();
+        const currencyListKeys = _.keys(currencyList);
```

Comment:
```suggestion
        const currencyKeyList = _.keys(currencyList);
```

Reply:
I think the initial name makes more sense, so I'll keep it